src/Stateless/EntryActionBehaviour.cs

```diff
@@ -31,6 +30,9 @@ public Sync(Action<Transition, object[]> action, string actionDescription) : bas
                     _action = action;
                 }
 
+                internal override string MethodName {  get { return _action.TryGetMethodName(); } }
```

Comment:
Since the class itself is internal (via the parent) these could just be `public`.